src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json

```diff
@@ -17,6 +18,8 @@
         "Viewset": "$.Parameters.Viewset",
         "GUIOption": "-nogui",
```

Comment:
In VC, do we ever want gui? is it a different performance test? If not we can remove this option as well.